setup.py

```diff
@@ -75,8 +75,14 @@
 mpi_root = None if no_dist else os.environ['MPIROOT']
 dpcpp = True if 'DPCPPROOT' in os.environ else False
 dpcpp_root = None if not dpcpp else os.environ['DPCPPROOT']
-dpctl = True if dpcpp and 'DPCTLROOT' in os.environ else False
-dpctl_root = None if not dpctl else os.environ['DPCTLROOT']
+
+try:
+    import dpctl
+    dpctl_available = dpctl.__version__ >= '0.14'
+except ImportError:
+    dpctl_available = False
```

Comment:
This code is duplicated 3 times. Please, move it to separate function in separate PR